.config/systemd/user/get-and-send-to-graphite:minutely@.timer

```diff
@@ -5,5 +5,9 @@ Description=Run get-and-send-to-graphite@%i each minute
 # OnUnitActiveSec is broken: https://github.com/systemd/systemd/issues/6680
 OnCalendar=minutely
 
+# randomize start to avoid using too much ressources
+AccuracySec=1s
+RandomizedDelaySec=20s
```

Comment:
This might still allow multiple timers to run at the same time. How many are allowed now by systemd?

Reply:
there is no limit that I know of. from what I understood, it is the number of subprocess that gets capped by systemd, so yeah, I some cases, it might have multiple services running at the same time, but hopefully, randomness at each launch and grafana leniency will make it transient enough.

if we want to only have a single service running at the same time, we would need to do timeslicing by hand (thus adding a timer per timeslice), which makes it way more cumbersome to manage and will probably not change much when things go haywire (what if a service takes longer to run than expected? the next timer will start anyway)

Reply:
I was just doing the 'birthday paradox' calculation: sqrt(20) is about 4, so if there are 4 processes that get 'mapped' to 1s intervals, there is a 50% chance that at least 2 are mapped to the same slice. Whereas if you map them to `1us` timeslices, and the processes are less than 1 second, you increase the slots available and reduce the probability that two processes start during the same slot.

From what I see, there are 
- 2 processes take around 1.5s
- 4 processes take around 400ms

So to best map these 6 processes, we'd need 6^2 slots to have a 50% of chance that there is no collision. More slots are better.

This is why I think `AccuracySec=1us` might increase the available slots in a usable manner.

Same for the 30m timer: `AccuracySec=1s` increases the available slots for 30s processes.